--- /omd/versions/default/skel/etc/apache/apache-own.conf.bak	2015-12-03 12:52:43.000000000 +0100
+++ /omd/versions/default/skel/etc/apache/apache-own.conf	2016-02-01 05:55:45.008616754 +0100
@@ -11,6 +11,9 @@
   ProxyRequests Off
   ProxyPreserveHost On
 
+  # Indicate when the site was accessed via HTTPS
+  RequestHeader set X-Forwarded-Proto https env=HTTPS
+
   # Include file created by 'omd config', which 
   # sets the TCP port of the site local webserver
   Include ###ROOT###/etc/apache/proxy-port.conf
