--- /omd/versions/1.2.6p15.cre/bin/omd.bak    2015-12-03 12:52:37.000000000 +0100
+++ /omd/versions/1.2.6p15.cre/bin/omd        2016-01-07 16:34:09.000555768 +0100
@@ -2141,8 +2141,7 @@
     # non RPM/DEB installations (FIXME: find a more elegant
     # solution later)
     gid = group_id('omd')
-    for plugin in [ 'nagios/plugins/check_icmp', 'nagios/plugins/check_dhcp',
-                    'cmc/icmpsender', 'cmc/icmpreceiver' ]:
+    for plugin in [ 'nagios/plugins/check_icmp', 'nagios/plugins/check_dhcp' ]:
         path = "/omd/versions/%s/lib/%s" % (OMD_VERSION, plugin)
         os.chown(path, -1, gid)
         os.chmod(path, 04750)
